rspec-mode.el

```diff
@@ -628,11 +628,14 @@ file if it exists, or sensible defaults otherwise."
           ;; 1.2.0+
           (let* ((path (or (getenv "XDG_RUNTIME_DIR") temporary-file-directory))
                  (ruby-version (shell-command-to-string "ruby -e 'print RUBY_VERSION'"))
-                 (application-id (md5 (concat ruby-version root))))
+                 (application-id (md5 (concat ruby-version root)))
+                 (full-application-id (md5 (concat ruby-version (expand-file-name root)))))
```

Comment:
We should probably compute `full-application-id` on the last line, when no other cases match.